sources/packages/backend/libs/sims-db/src/entities/assessment-trigger.type.ts

```diff
@@ -37,3 +37,9 @@ export enum AssessmentTriggerType {
    */
   StudentAppeal = "Student appeal",
 }
+
+export enum StudentAssessmentStatus {
```

Comment:
I am not sure if this enum belongs to the entity layer, but it potentially will, so I do not see a problem keeping it at this layer. Please move it to a file and add the proper comments.
While adding the comments please base them on the below logic.
![image](https://github.com/bcgov/SIMS/assets/61259237/cbcc00e6-ee0f-49c7-bef9-daac64cb6d13)